### DISCUSSION TITLE
运行prepublish时

### DISCUSSION THREAD

**Discussion by sunzhangchang** [napi-rs/napi-rs#1083] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/52880063/156325540-f9e327b3-6680-40cc-bdc5-eec8a496a825.png)

这是为什么？

**Comment by Brooooooklyn** [comment, marked as answer]
如果是在非 Github Actions 环境内发布，需要加上 `--skip-gh-release` 。下个版本 cli 会自动探测并兼容其他环境
